src/style.css

```diff
@@ -0,0 +1,148 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  font-style: book;
```

Comment:
Styles like 'book' and 'heavy' don't exist. Remove it
check [this link](https://developer.mozilla.org/en-US/docs/Web/CSS/font-style)